frontend/packages/console-shared/src/utils/pod-ring-utils.ts

```diff
@@ -45,19 +45,22 @@ export const podRingLabel = (
 ): PodRingLabelType => {
   const {
     spec: { replicas },
-    status,
+    status: { currentNumberScheduled, desiredNumberScheduled, readyReplicas },
     kind,
   } = obj;
-  const { availableReplicas } = status || {};
 
-  const isPending = (pods?.length === 1 && pods[0].status?.phase === 'Pending') || replicas;
-  const pluralize = replicas > 1 || replicas === 0 ? 'pods' : 'pod';
+  const currentPodCount = readyReplicas || currentNumberScheduled;
+  const desiredPodCount = replicas || desiredNumberScheduled;
+  const isPending =
+    (pods?.length === 1 && pods[0].status?.phase === 'Pending') ||
+    (desiredPodCount === 1 && desiredPodCount !== currentPodCount);
+  const pluralize = currentPodCount !== 1 ? 'pods' : 'pod';
```

Comment:
should use pluralize(currentPodCount, 'pod') method from patternfly/react-core and check for isPending before calling pluralize
something like `!isPending && currentPodcount !== 0 && pluralize(currentPodCount, 'pod')`

Reply:
I updated the code to use `pluralize(currentPodCount, 'pod', 'pods')`
i used `readyReplicas` instead of `availableReplicas` for the podRingText, which seemed to solve the problem. PTAL